frontend/src/components/Playback/Playback.js

```diff
@@ -81,27 +103,57 @@ const Playback = ({
     // Play audio
     const playAudio = useCallback(
         (index) => {
-            // Only initialize webaudio if section is local            
-            let latency = 0;      
-            // Store player index
-            setPlayerIndex(index);
-            // Determine if audio should be played
-            if (playConfig.mute) {
-                setPlayerIndex(-1);
-                audio.pause();
-                return;
-            }
-            // Volume 1
-            audio.setVolume(1);
+            let latency = 0;
+            if (playConfig.play_method === 'BUFFER' && !playConfig.external_audio) {
+                console.log('Play buffer (local)');                
+                // Determine latency for current audio device
+                latency = webAudio.getTotalLatency();
+                // Store player index
+                setPlayerIndex(index);
+                // Determine if audio should be played
+                if (playConfig.mute) {
+                    setPlayerIndex(-1);
+                    webAudio.stopBuffer();
+                    return;
+                }
+                // Volume 1
+                // audio.setVolume(1);
+    
+                // Cancel active events
+                cancelAudioListeners();           
+    
+                // Play audio
+                // audio.playFrom(Math.max(0, playConfig.playhead || 0));
+                
+                webAudio.playBuffer(sections[index].id);
+                // listen for active audio events
+                activeAudioEndedListener.current = webAudio.listenOnce("ended", onAudioEnded);                
+            } else {
+                console.log('Play HTML audo')
+                // Only initialize webaudio if section is local            
+                if (!playConfig.external_audio) {                    
+                    latency = webAudio.initWebAudio();
+                }
+                // Store player index
+                setPlayerIndex(index);
+                // Determine if audio should be played
+                if (playConfig.mute) {
+                    setPlayerIndex(-1);
+                    audio.pause();
+                    return;
+                }
```

Comment:
There is a lot of code repeated in this if / else condition. Once you remove the log statements, perhaps it would also be good to outfactor the differences between html / web audio into a utils file, and then deal with the differences there, so you can call, for instance, the utils method `pauseAudio(playMethod)`, and this method will handle the logic of whether to call pause via the audio element or the webAudio API.